include/albatross/src/eigen/serializable_ldlt.hpp

```diff
@@ -88,10 +97,10 @@ class SerializableLDLT : public LDLT<MatrixXd, Lower> {
    * Computes the product of the square root of A with rhs,
    *   D^{-1/2} L^-1 P rhs
    */
-  template <class Rhs>
-  Eigen::MatrixXd sqrt_solve(const MatrixBase<Rhs> &rhs) const {
+  template <class Rhs> Eigen::MatrixXd sqrt_solve(const Rhs &rhs) const {
     return diagonal_sqrt_inverse() *
-           this->matrixL().solve(this->transpositionsP() * rhs);
+           this->matrixL().solve(this->transpositionsP() *
+                                 Eigen::MatrixXd(rhs));
```

Comment:
will this end up copying `rhs`?

Reply:
That's a good point, I'll figure something else out.

Reply:
Copy removed with no apparent issue.

Reply:
Are we sure that `Eigen::MatrixXd(rhs)` isn't copying?

Reply:
Oh, I'm not sure why this is still in the change.  I thought I removed it.  I'll make sure it's gone.